pkg/networkservice/chains/nsmgr/suite_test.go

```diff
@@ -510,6 +537,35 @@ func (s *nsmgrSuite) Test_PassThroughLocalUsecase() {
 	}
 }
 
+func connectServer(ctx context.Context, requestCount *int, index int, t *testing.T, connCtx *networkservice.ConnectionContext) networkservice.NetworkServiceServer {
+	return connect.NewServer(ctx,
+		client.NewClientFactory(
+			client.WithName(fmt.Sprintf("nsc-%v", uuid.New().String())),
+			client.WithAdditionalFunctionality(
+				checkconnection.NewClient(t, func(t *testing.T, conn *networkservice.Connection) {
+					assert.True(t, proto.Equal(connCtx, conn.Context))
```

Comment:
I think we need to have two checks:
1. Check that contextContext is updated on backward(before translation client.)
2. Check that contextContext is updated on forward ( after translation client.)